src/index.html

```diff
@@ -1,13 +1,368 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>myBike</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;700&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="shortcut icon" href="./images/favicon.svg" type="image/x-icon">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#" class="logo">
+              <img
+              class="logo__image"
+              src="./images/MyBiKE.svg"
+              alt="MyBiKE">
```

Comment:
Fix code style on all file

```suggestion
              <img
                class="logo__image"
                src="./images/MyBiKE.svg"
                alt="MyBiKE"
              >
```

Reply:
fixed.